src/helpers.js

```diff
@@ -10,8 +10,78 @@ export default class Helper {
   formHandler(e) {
     e.preventDefault();
     utils.add();
-    document.querySelector('.todo-list ul').innerHTML = '';
-    document.querySelector('.todo-list ul').insertAdjacentHTML('beforeend', utils.render());
     utils.clear();
+    this.checkHandler();
+    this.handleFocus();
+  }
+
+  /**
+   * @function display - this helps to display the list on load
+   */
+  display() {
+    const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+    tasks.forEach((item) => {
+      const { description, index } = item;
+      utils.render(description, index);
+      document.querySelector('#todo').focus();
+      this.checkHandler();
+    });
+  }
+
+  /**
+   * @function focusHandler - handles the focus on the inputs
+   * @param {Event Object} e - The current event
+   */
+  focusHandler(e) {
+    const item = e.currentTarget;
+    const child = item.children;
+    if (document.querySelector('li.list.focus')) {
+      const parent = document.querySelector('li.list.focus');
+      parent.children[2].classList.remove('focus');
+      parent.children[3].classList.remove('d-none');
+      parent.children[4].classList.add('d-none');
+      parent.classList.remove('focus');
+    }
+    child[2].focus();
+    utils.tog('add', 'focus', child[2]);
+    utils.tog('add', 'd-none', child[3]);
+    utils.tog('rem', 'd-none', child[4]);
+    item.classList.add('focus');
+  }
+
+  /**
+   *
+   * @function changeHandler - handles the change on the checkbox
+   * @param {Event Object} e - The current event
+   */
+  changeHandler(e) {
+    const item = e.target;
+    const parent = item.parentElement;
+    const child = parent.children;
+    utils.tog('add', 'd-none', item);
+    utils.tog('add', 'strike', child[2]);
+    utils.tog('rem', 'd-none', child[1]);
+    utils.change(item.dataset.id, true);
+    child[1].addEventListener('click', (e) => {
+      utils.tog('rem', 'strike', child[2]);
+      utils.tog('add', 'd-none', e.target);
+      utils.tog('rem', 'd-none', item);
+      item.checked = false;
+      utils.change(item.dataset.id, false);
+    });
+  }
```

Comment:
- [ ] Let's make sure that when a task is toggled to completed or uncompleted, the state of the task is also stored in the LocalStorage. Right now, if I change a task to completed and try to reload the page, every task is set to uncompleted again.

Update: I checked the LocalStorage, and the state of each task is correctly being stored. So the bug occurs when you render the tasks.

![image](https://user-images.githubusercontent.com/79114233/143918087-42fcd766-4a5d-42d8-bd71-027aa24ae588.png)